festim/exports/trap_density_xdmf.py

```diff
@@ -16,13 +16,27 @@ def __init__(self, trap, **kwargs) -> None:
 
         self.trap = trap
 
-    def write(self, t):
+    def write(self, t, dx, log_level):
```

Comment:
I wouldn't have it as an argument.

Imagine if we want to have this everywhere then you'd have to pass log_level literally everywhere.

Maybe let's forget about this for this PR.

What may be a solution is to have log_level an attribute of FESTIM.

Instead of:

```python
my_model.log_level = 20
```

we'd have

```python
import festim

festim.log_level = 20
```

That way, log_level can be accessed from anywhere.

Reply:
This would be a nice solution, i'll remove it from the function